aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -982,6 +982,34 @@ Tensor hann_window(
       window_length, periodic, /*alpha=*/0.5, /*beta=*/0.5, options);
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~ kaiser_window ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tensor kaiser_window(int64_t window_length, double beta, const TensorOptions& options) {
+  return native::kaiser_window(window_length, beta, /*periodic=*/true, options);
+}
+
+Tensor kaiser_window(
```

Comment:
To be consistent with PyTorch's windowing functions the name is `kaiser_window`, like you have here, but the order of arguments would be `window_length, periodic, beta`. See, for example, the Hamming window documentation which has alpha and beta terms:

https://pytorch.org/docs/master/generated/torch.hamming_window.html?highlight=window#torch.hamming_window

To be consistent with NumPy we can add a `kaiser` function later that has a NumPy-compatible signature.